saas/api/subscriptions.py

```diff
@@ -1072,3 +1073,5 @@ def post(self, request, *args, **kwargs):
         signals.subscription_request_accepted.send(sender=__name__,
             subscription=self.subscription,
             request_key=request_key, request=self.request)
+
+        return Response({})
```

Comment:
The API documentation shows that the API should return a `ProvidedSubscriptionSerializer`-formatted result. i.e:

```
    @extend_schema(request=None, responses={
      200: OpenApiResponse(ProvidedSubscriptionSerializer)})
```

So the better fix would be to use the following statement:

```
        serializer = self.get_serializer(instance=self.subscription)
        return Response(serializer.data)
```